### ISSUE TITLE
Support Raspberry Pi

### ISSUE THREAD

**Issue by nuttyartist** [nuttyartist/notes#68]
We need someone that has one and can help make Notes supported on it.

**Comment by theshadowx** [comment]
Raspberry Pi uses Raspbian which is Debian, so what should be done is to compile the app for arm processor. 

One of the solutions is to use Docker so It would be possible to compile and generate the executable without having Raspberry.

I have created before an image of Raspbian (wheezy) with Qt 5.5 for Raspberry https://hub.docker.com/r/theshadowx/rpi-qt/

**Comment by nuttyartist** [comment]
oh cool, so all we need is to compile it on the Docker container, and package it as a .deb file like we normally do?

**Comment by theshadowx** [comment]
yes, there is also another way to generate the package is by writing a script that will go through all the steps needed to generate a package inside the container and as result you'll have the package ready to be used

**Comment by nuttyartist** [comment]
Great, I'm adding this to #13.

**Comment by hmsjy2017** [comment]
I think the better solution is to use Travis-CI or Azure Pipelines to build, because they have Linux/ARM64 runners.
